lms/djangoapps/courseware/views.py

```diff
@@ -726,6 +726,23 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        return "{}{}".format(currency_symbol, price)
```

Comment:
Not all currencies have the symbol displayed before the value. This could be a good candidate for i18n -- although I'm not sure if we're doing i18n for currency formatting. @sarina, any thoughts?